test/test_train_mp_mnist.py

```diff
@@ -73,7 +76,7 @@ def _train_update(device, step, loss, tracker, epoch, writer):
 
 
 def train_mnist(flags, **kwargs):
-  if flags.ddp:
+  if flags.ddp or flags.pjrt_distributed:
```

Comment:
Ah interesting... Do we need a dedicated flag still, or can we just also check for torchrun some other way? I saw [dist.is_torchelastic_launched](https://pytorch.org/docs/stable/distributed.html) elsewhere in the codebase: https://github.com/pytorch/xla/blob/83778f0/torch_xla/_internal/rendezvous.py#L20